src/containers/checkout/CheckoutOverview.tsx/CheckoutIntroduction.tsx

```diff
@@ -9,39 +9,59 @@ import FallbackImage from 'src/images/hotel-fallback.webp';
 
 export const CheckoutIntroduction = () => {
   const { bookingMode, bookingInfo } = useCheckout();
+
+  if (!bookingInfo || !bookingInfo.pricing || !bookingInfo.accommodation) {
+    // don't render anything without pricing info, or accommodation info
+    return null;
+  }
+
   const isGroupMode = bookingMode === 'group' ? true : false;
 
-  if (!bookingInfo?.pricing) return null;
+  let googleMapsLink = '';
+  if (
+    bookingInfo.accommodation.location &&
+    bookingInfo.accommodation.location.coordinates &&
+    bookingInfo.accommodation.location.coordinates.length === 2
+  ) {
+    const lonCoord = bookingInfo.accommodation.location.coordinates[0];
+    const latCoord = bookingInfo.accommodation.location.coordinates[1];
+
+    googleMapsLink = `https://www.google.com/maps/search/?api=1&query=${latCoord},${lonCoord}`;
+  }
 
   const formattedOfferPrice = formatPrice(
-    utils.parseEther(bookingInfo.pricing?.offerCurrency.amount.toString()),
-    bookingInfo.pricing?.offerCurrency.currency
+    utils.parseEther(bookingInfo.pricing.offerCurrency.amount.toString()),
+    bookingInfo.pricing.offerCurrency.currency
   );
 
-  const title = isGroupMode
+  const paymentValue = isGroupMode
     ? `The refundable deposit is ${formattedOfferPrice}`
     : `Your payment value is ${formattedOfferPrice}`;
 
   const formattedUsdPrice =
-    bookingInfo.pricing?.usd &&
-    formatPrice(utils.parseEther(bookingInfo.pricing.usd.toString()), 'USD');
-  const subTitle = `Equivalent to ${formattedUsdPrice}`;
+    bookingInfo.pricing.usd &&
+    formatPrice(utils.parseEther(bookingInfo.pricing.usd.toString()));
 
   const showUSDPrice =
-    formattedUsdPrice && bookingInfo.pricing?.offerCurrency.currency != 'USD';
+    formattedUsdPrice && bookingInfo.pricing.offerCurrency.currency != 'USD';
 
-  const accommodationName = bookingInfo?.accommodation?.name;
+  const accommodationName = bookingInfo.accommodation.name;
   const accommodationImage = useMemo(() => {
-    if (bookingInfo?.accommodation) {
+    if (bookingInfo.accommodation) {
       return sortByLargestImage(bookingInfo.accommodation.media)[0];
     }
   }, [bookingInfo]);
 
   return (
-    <Box mb={{ xs: 3, lg: 5 }}>
-      <Typography>{title}</Typography>
-      {showUSDPrice && <Typography>{subTitle}</Typography>}
-
+    <Box
+      mb={{ xs: 3, lg: 5 }}
+      style={{
+        border: '0.1em solid #637381',
+        borderRadius: '0.3em',
+        padding: '0.6em',
```

Comment:
i would stick to the padding of the theme=> if you pass a number it will automatically get the padding from the theme. More info here: https://mui.com/system/spacing/